doc/doxygen/headers/glossary.h

```diff
@@ -1485,7 +1485,16 @@
  *   paper lays out the motivation for this pattern and then proposes different ways
  *   of implementing it. It also compares the performance of different implementations.
  *
- *   The paper is currently in preparation.
+ * The full reference for this paper is as follows:
+ * @code
+@Article{TKB16,
```

Comment:
Is there a reason you are not including " * " in front of these lines? Doxygen should rip them out I think.